docs/How to setup mysql plugin.md

```diff
@@ -0,0 +1,9 @@
+# How to setup Mysql plugin for OMS agent
+
+How to enable mySQL plugin to collect mySQL server stats and perf data
+* Install mySQL server on the machine
+* Machine has to have libmysqlclient.so installed(can be downloaded and installed through mysql-devel)
```

Comment:
Is this for customers, or for us? This is VERY brief, and I'd like to see more info here. HOW do you install it on RPM based systems? (yum) How do you install it on DPKG systems (apt-get).

Also, getting libmysqlclient.so is a bit harder on Ubuntu, so you might want to mention the kit to include from apt-get to grab this.